src/helm/benchmark/window_services/luminous_window_service.py

```diff
@@ -0,0 +1,119 @@
+from abc import abstractmethod
```

Comment:
As we get more and more models, it seems like these window service classes have a ton of boilerplate - stepping back, do you think there's anything we can do about it?

Reply:
Sorry, I copied the code from `LocalWindowService` (the parent of this class) and forgot to remove the code I didn't need.

Reply:
I still have some questions for them regarding the max context length and `echo`, but  I will finalize after I follow up.